justfile

```diff
@@ -22,6 +22,10 @@ test_async_std_all:
   echo Testing with async std executor
   cargo test  --features=full-ci --lib --bins --tests --benches --workspace --no-fail-fast -- --test-threads=1
 
+_test_consensus:
```

Comment:
Forgot to comment on this earlier--I think we could remove the underscore like we did to `test_basic`.